src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/ListingService.java

```diff
@@ -144,6 +154,18 @@ public ListingData processListingHearingsRequest(ListingDetails listingDetails,
         }
     }
 
+    private void populateScottishVenues(ListingData listingData) {
+        if ("Aberdeen".equals(listingData.getManagingOffice())) {
```

Comment:
You can use the TribunalOffice enum rather than hard code office names
`TribunalOffice.ABERDEEN.getOfficeName()`

Reply:
Code refactored